src/containers/StartupProjects/StartupProject.js

```diff
@@ -27,21 +27,26 @@ export default function StartupProject() {
           >
             {bigProjects.subtitle}
           </p>
-          <div className="startup-projects-main">
-            <div className="startup-project-text">
+
+          <div className="project-cards-div">
               {bigProjects.projects.map((project,i) => {
                 return (
-                  <div
-                    key={i}
-                    className="saaya-health-div"
-                    onClick={() => openProjectInNewWindow(project.link)}
-                  >
-                    <img alt="Saad Working" src={project.image}></img>
+                  <div className={isDark ? "dark-mode project-card" : "project-card"} 
+                  onClick={() => openProjectInNewWindow(project.link)}>
+                  <div className="project-image-div">
```

Comment:
Similarly here, use `project-image` for classname

Reply:
Also, is this div flexible enough to look 'normal' and won't break if the user provides no image to the card. Since we don't have a preview deployment yet, I'd appreciate a screenshot of the behavior to be sure!